general/1_terminal/README.md

```diff
@@ -0,0 +1,35 @@
+# homework 1
+
+
+![](images/task1.png)
+
+proof that i did task 0
+
+## task 1
+
+'ls -l | grep -i zoola'
+
+## task 2
+Shell scripts are run inside a subshell, and each subshell has its own concept of what the current directory is. The `cd` succeeds, but as soon as the subshell exits, you're back in the interactive shell and nothing ever changed there.
+[This link helped me](https://stackoverflow.com/questions/255414/why-cant-i-change-directories-using-cd-in-a-script)
+## task 3
+
+```
```

Comment:
```suggestion
```sh
```

A nice Markdown trick you can use is to specify the language of the code block to enable syntax highlighting.

You can learn more [on this GitHub doc page](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks).

This is not mandatory, but nice to know IMO.

For example, check out this elm code snippet:


```elm
import Browser
import Html exposing (Html, button, div, text)
import Html.Events exposing (onClick)

main =
  Browser.sandbox { init = 0, update = update, view = view }

type Msg = Increment | Decrement

update msg model =
  case msg of
    Increment ->
      model + 1

    Decrement ->
      model - 1

view model =
  div []
    [ button [ onClick Decrement ] [ text "-" ]
    , div [] [ text (String.fromInt model) ]
    , button [ onClick Increment ] [ text "+" ]
    ]
```